ts/Extensions/Annotations/Controllables/Controllable.ts

```diff
@@ -190,6 +190,9 @@ abstract class Controllable implements ControlTarget {
         // eslint-disable-next-line @typescript-eslint/no-unused-vars
         _parentGroup?: SVGElement
     ): void {
+        if (this.options.className && this.graphic) {
+            this.graphic.addClass(this.options.className);
+        }
```

Comment:
Good, now remove other `addClass()`, in each Controllable class, so we don't duplicate (code and classname)